conanfile.py

```diff
@@ -154,11 +176,27 @@ def configure_cmake(self):
             "ON" if self.options.opengl3_renderer else "OFF"
         cmake.definitions["OGRE_BUILD_RENDERSYSTEM_GL"] = \
             "ON" if self.options.opengl_renderer else "OFF"
-        cmake.definitions["OGRE_BUILD_RENDERSYSTEM_GLES2"] = \
-            "ON" if self.options.opengles_renderer else "OFF"
+        #cmake.definitions["OGRE_BUILD_RENDERSYSTEM_GLES2"] = \ # Getting build error: "Cannot open include file 'EGL/egl.h'"
```

Comment:
See comments above regarding this. I think it is best to leave it in the recipe and rather add system_requirements.